Libplanet.Action/ActionEvaluator.cs

```diff
@@ -555,8 +554,6 @@ internal IWorld PrepareInitialDelta(HashDigest<SHA256>? stateRootHash)
                     exception: evaluation.Exception);
                 committedEvaluations.Add(committedEvaluation);
 #pragma warning restore SA1118
-
-                trie = evaluation.OutputState.Trie;
```

Comment:
is it intended?

Reply:
It seems like an artifact from the past? It isn't really being used.

Reply:
Would it be a problem if you didn't put the previous `ITrie` into the next state?

Reply:
This simply converts `IActionEvaluation`s to `ICommittedActionEvaluation`s. Each `IActionEvaluation` already has input and output `ITrie`s. I can't remember exactly why reassigning next `ITrie` was used at the moment. 🙄

Reply:
Note that `trie` variable was not being referenced in the first place. 🙃